cmd/config_labels.go

```diff
@@ -0,0 +1,286 @@
+the local machine.
+`,
+	SuggestFor: []string{"ad", "create", "insert", "append"},
+	PreRunE:    bindEnv("path"),
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+		function, err := initConfigCommand(args)
+		if err != nil {
+			return
+		}
+
+		return runAddLabelsPrompt(cmd.Context(), function)
+	},
+}
+
+var configLabelsRemoveCmd = &cobra.Command{
+	Use:   "remove",
+	Short: "Remove labels from the function configuration",
+	Long: `Remove labels from the function configuration
+
+Interactive prompt to remove labels from the function project in the current
+directory or from the directory specified with --path.
+`,
+	SuggestFor: []string{"del", "delete", "rmeove"},
+	PreRunE:    bindEnv("path"),
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+		function, err := initConfigCommand(args)
+		if err != nil {
+			return
+		}
+
+		return runRemoveLabelsPrompt(function)
+	},
+}
+
+func listLabels(f fn.Function) {
+	if len(f.Labels) == 0 {
+		fmt.Println("There aren't any configured labels")
+		return
+	}
+
+	fmt.Println("Configured labels:")
+	for _, e := range f.Labels {
+		fmt.Println(" - ", e.String())
+	}
+}
+
+func runAddLabelsPrompt(ctx context.Context, f fn.Function) (err error) {
+
+	insertToIndex := 0
+
+	// SECTION - if there are some labels already set, choose the position of the new entry
+	if len(f.Labels) > 0 {
+		options := []string{}
+		for _, e := range f.Labels {
+			options = append(options, fmt.Sprintf("Insert before:  %s", e.String()))
+		}
+		options = append(options, "Insert here.")
+
+		selectedLabel := ""
+		prompt := &survey.Select{
+			Message: "Where do you want to add the label?",
+			Options: options,
+			Default: options[len(options)-1],
+		}
+		err = survey.AskOne(prompt, &selectedLabel)
+		if err != nil {
+			if err == terminal.InterruptErr {
+				return nil
+			}
+			return
+		}
+
+		for i, option := range options {
+			if option == selectedLabel {
+				insertToIndex = i
+				break
+			}
+		}
+	}
+
+	// SECTION - select the type of label to be added
+	selectedOption := ""
+	const (
+		optionLabelValue = "Label with a specified value"
+		optionLabelLocal = "Value from a local environment variable"
+	)
+	options := []string{optionLabelValue, optionLabelLocal}
+
+	err = survey.AskOne(&survey.Select{
+		Message: "What type of label do you want to add?",
+		Options: options,
+	}, &selectedOption)
+	if err != nil {
+		if err == terminal.InterruptErr {
+			return nil
+		}
+		return
+	}
+
+	newTuple := fn.Tuple{}
+
+	switch selectedOption {
+	// SECTION - add new label with the specified value
+	case optionLabelValue:
+		qs := []*survey.Question{
+			{
+				Name:   "name",
+				Prompt: &survey.Input{Message: "Please specify the label name:"},
+				Validate: func(val interface{}) error {
+					return utils.ValidateLabelName(val.(string))
+				},
+			},
+			{
+				Name:   "value",
+				Prompt: &survey.Input{Message: "Please specify the label value:"},
```

Comment:
Shouldn't be the value validated as well?
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set